hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/TestBlockDeletingService.java

```diff
@@ -155,24 +178,29 @@ private void createToDeleteBlocks(ContainerSet containerSet,
           BlockData kd = new BlockData(blockID);
           List<ContainerProtos.ChunkInfo> chunks = Lists.newArrayList();
           for (int k = 0; k < numOfChunksPerBlock; k++) {
+            final String chunkName = String.format("block.%d.chunk.%d", j, k);
+            final long offset = k * blockLength;
             ContainerProtos.ChunkInfo info =
                 ContainerProtos.ChunkInfo.newBuilder()
-                    .setChunkName(blockID.getLocalID() + "_chunk_" + k)
+                    .setChunkName(chunkName)
                     .setLen(blockLength)
-                    .setOffset(0)
+                    .setOffset(offset)
                     .setChecksumData(Checksum.getNoChecksumDataProto())
                     .build();
             chunks.add(info);
+            ChunkInfo chunkInfo = new ChunkInfo(chunkName, offset, blockLength);
+            ChunkBuffer chunkData = buffer.duplicate(0, (int) blockLength);
+            chunkManager.writeChunk(container, blockID, chunkInfo, chunkData,
+                WRITE_STAGE);
+            chunkManager.writeChunk(container, blockID, chunkInfo, chunkData,
+                COMMIT_STAGE);
           }
           kd.setChunks(chunks);
           metadata.getStore().getBlockDataTable().put(
                   deleteStateName, kd);
           container.getContainerData().incrPendingDeletionBlocks(1);
         }
-
         container.getContainerData().setKeyCount(numOfBlocksPerContainer);
-        container.getContainerData().setBytesUsed(
-            blockLength * numOfBlocksPerContainer);
```

Comment:
Can we also change the value for OzoneConsts.CONTAINER_BYTES_USED updated in table? 
The new value should be `chunkLength * numOfChunksPerBlock * numOfBlocksPerContainer`.

Reply:
Value changed.